snippets/reject.md

```diff
@@ -0,0 +1,17 @@
+### reject
+
+Returns the elements of an array for which the passed function returns `false`.
+
+Use `Array.filter()` and the not operator (`!`) to return the elements in the array for which `fn` returns `false`.
+
+```js
+const reject = (arr, fn) => arr.filter((e,i,arr) => !fn(e,i,arr));
```

Comment:
breaks binding of context provision of the filter method

Reply:
Meaning `this` is broken in there?

Reply:
I can't think of a single use case where I'd use this over filter with a proper predicate. If someone is using boolean logic and can't design a proper predicate, what use would this function be for them?

Reply:
I only implemented this because lodash has it and, to be honest, I felt the same. Should we just can this and mark in #100 to note we are not implementing?

Reply:
We also have `negate` which should be enough with composition, let's kill this one then.

Reply:
the awkward part with negate is that it also kills the bindable, and does exactly what this snippet is with composition:
```js
const reject = (f, a) => a.filter(negate(f)) // kills bindable unless negate is implemented as a function versus an arrow function
```

Reply:
In all honesty these start to get confusing and both are terribly niche and forgettable. I think we shall keep `negate` just as an example (not a great one, but still) and remove `reject` and any other methods that are just boolean mumbo-jumbo. How difficult is it for a developer to add a `!` anyways?

Reply:
Lodash has a lot of fluff methods that are not useful. It's better to avoid this and a few others such as the `flattenMap` ones in Object.